refarch-frontend/src/locales/README.md

```
@@ -0,0 +1,73 @@
+       paragraph: "Some text here..."
+     ```
+
+3. **Consistent Section Prefixes**
+   - Maintain distinct top-level sections: `common`, `app`, `views`, `components`, `domain`, etc.
+   - This consistency makes it easier to quickly locate strings in a large codebase.
+
+## Special Character Handling
+
+- When using special or reserved YAML characters (e.g., `@`, `'`, `"`, `{`), ensure they are properly quoted if needed.
+- Complex strings may require double quotes to avoid parsing or lint errors, for example:
+  ```yaml
```

Markdownlint (0.37.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)